### PR TITLE
NCP-3341 Added labels and validation messages for fields

### PR DESCRIPTION
**Pull request by adityakulkarni27**
This is in accordance with - [https://github.com/mileweb/ngportal/pull/2440]()

### CODE REVIEW

File: en_US.json [hunk 1]
```diff
@@ -823,11 +823,15 @@
...
 	"error.origin.auto.detect.popup.msg": "We could not detect a server based on your hostname. Please manually enter origin settings.",
 	"error.origin.auto.detect.popup.title": "Auto Detection Failed",
+	"error.origin.failureThresholdLabel.range": "The Failure Threshold value must be in the range [1, 999]",
+	"error.origin.failureThresholdLabel.required": "Failure Threshold value is required",
 	"error.origin.invalid.domain.name": "Must be a valid domain name or a variable starts with '$'.",
 	"error.origin.invalid.keepalive.timeout": "The Keep-Alive Timeout value must be in the range [5, 600]",
 	"error.origin.invalid.tlsSessionTimeoutDefault": "Invalid TLS Session Timeout value.",
 	"error.origin.invalid.tlsSessionTimeoutRange": "The TLS Session Timeout value must be in the range [300, 86400]",
 	"error.origin.select.required": "Please select an origin.",
+	"error.origin.timeout.range": "The Keep-Alive Timeout value must be in the range [1, 1200]",
```
Comment (chiraags) [inline en_US.json:833]: This timeout it specific to peerFailureTimeout and not keep-alive timeout.

Comment (adityakulkarni27) [reply en_US.json]: @chiraags I have updated it in the next commit. Please let me know if there are any more changes required!

File: en_US.json [hunk 2]
```diff
@@ -823,11 +823,15 @@
 	"error.origin.authentication.awss3.secretkey.length": "The AWS S3 secret key must be at most 4096 characters.",
 	"error.origin.auto.detect.popup.msg": "We could not detect a server based on your hostname. Please manually enter origin settings.",
 	"error.origin.auto.detect.popup.title": "Auto Detection Failed",
+	"error.origin.failureThresholdLabel.range": "The Failure Threshold value must be in the range [1, 999]",
+	"error.origin.failureThresholdLabel.required": "Failure Threshold value is required",
```
Comment (chiraags) [inline en_US.json:827]: "The Failure Threshold" to keep it consistent with the related message.

File: en_US.json [hunk 3]
```diff
@@ -823,11 +823,15 @@
...
 	"error.origin.invalid.domain.name": "Must be a valid domain name or a variable starts with '$'.",
 	"error.origin.invalid.keepalive.timeout": "The Keep-Alive Timeout value must be in the range [5, 600]",
 	"error.origin.invalid.tlsSessionTimeoutDefault": "Invalid TLS Session Timeout value.",
 	"error.origin.invalid.tlsSessionTimeoutRange": "The TLS Session Timeout value must be in the range [300, 86400]",
 	"error.origin.select.required": "Please select an origin.",
+	"error.origin.timeout.range": "The Peer Failure Timeout value must be in the range [1, 1200]",
```
Comment (chiraags) [inline en_US.json:833]: "The Timeout value must be in the range"  since this field is a part of the peerFailureTimeout setting.

Comment (adityakulkarni27) [reply en_US.json]: @chiraags I have made the changes in the new commit.

**Review by chiraags — APPROVED** [review]
Good to merge.

**Pull request merged**